### ISSUE TITLE
Does not work with Community Solid Server 1.0.0

### ISSUE THREAD

**Issue by nickform** [NoelDeMartin/media-kraken#14]
[Please explain what you were trying to do when this error appeared.]

I was experimenting with the Community Solid Server (just two commits after the v1.0.0 tag). It seems that CSS pods are created without a lot of the structure that came OOTB with pods at inrupt.net and I actually hit a couple of problems due to that before getting to this error. The first problem was that the app couldn't find a storage. Using https://penny.vincenttunru.com/, I added that property to my profile which then got me to another error because there was no private type index. I tried to address this by using Penny to see how this was set up in my inrupt.net pod and then to duplicate it in my CSS pod which worked to the extent that it changed the error message I got to the one I'm reporting here. After first hitting that, I manually created /blitzkrieg/movies/ using Penny but that did not help.

Stack trace:
```e@https://noeldemartin.github.io/media-kraken/js/0.60c393e8.worker.js?__WB_REVISION__=1ea26ee9042b4a85c3b0386d545dacc4:23:153141
e@https://noeldemartin.github.io/media-kraken/js/0.60c393e8.worker.js?__WB_REVISION__=1ea26ee9042b4a85c3b0386d545dacc4:23:269325
t/this.fetch/</<@https://noeldemartin.github.io/media-kraken/js/0.60c393e8.worker.js?__WB_REVISION__=1ea26ee9042b4a85c3b0386d545dacc4:23:273331
s/</<@https://noeldemartin.github.io/media-kraken/js/0.60c393e8.worker.js?__WB_REVISION__=1ea26ee9042b4a85c3b0386d545dacc4:23:272135
s/<@https://noeldemartin.github.io/media-kraken/js/0.60c393e8.worker.js?__WB_REVISION__=1ea26ee9042b4a85c3b0386d545dacc4:23:272240
s@https://noeldemartin.github.io/media-kraken/js/0.60c393e8.worker.js?__WB_REVISION__=1ea26ee9042b4a85c3b0386d545dacc4:23:271023

```

**Comment by NoelDeMartin** [comment]
Hi there! Thanks for opening an issue :).

I am aware of these problems, but I hadn't fixed them in this app yet. The first one should be fixed by adding `pim:storage` to your profile, and the second one should be fixed by creating a type index and declaring it in your profile as well.

However, I just tried these two fixes locally and it seems like there is another problem that I have to look into.

I don't think it should be too difficult to fix, but I'm going on holidays tomorrow so I don't want to risk publishing a broken version that won't be updated in a couple of weeks :sweat_smile:.

I will probably fix this early September, thanks for reporting!

**Comment by nickform** [comment]
Thanks. The app is awesome and I'm happily using it with my inrupt pod so there's no hurry for a fix from my perspective. Your ramen app did work btw. Have a great holiday. 👍

**Comment by NoelDeMartin** [comment]
This has been fixed with the `v0.10.0` release, make sure that you are using the latest version looking at the version number in the footer.

Let me know if you're having any further issues!

**Comment by nickform** [comment]
Just a quick note to say I tested this just now and it works seamlessly with CSS now. Thanks for fixing!
